doghouse/client/client.go

```diff
@@ -0,0 +1,65 @@
+package client
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	"github.com/haya14busa/reviewdog/doghouse"
+)
+
+const baseEndpoint = "https://review-dog.appspot.com"
+
+type DogHouseClient struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type DogHouseClient should have comment or be unexported